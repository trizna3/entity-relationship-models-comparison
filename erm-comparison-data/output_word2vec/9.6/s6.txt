life insurance-life insurance
persons-EMPTY
real estate types-EMPTY
account-customers;accounts
real estate insurance-real estate insurance
addresses-EMPTY
contract-contracts
CHANGE_CARDINALITY:S-accounts-customers
CONTRACT_11_ASSOCIATION:S-customers-accounts
