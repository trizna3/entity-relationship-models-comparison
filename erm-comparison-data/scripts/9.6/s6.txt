entity set customers {
};

entity set accounts {
	"name"
	"password"
};

entity set contracts {
	"from"
	"to"
	"monthly fee"
};

entity set "life insurance" {
	"death amount"
	"injury amount"
	"daily dose"
	"customer name"
	"customer surname"
	"other customer data"
	"insured name"
	"insured surname"
	"other insured data"
};

entity set "real estate insurance" {
	"crash insurance"
	"disaster insurance"
	"building address"
	"buiding type"
	"living area size"
	"cellar size"
};

association between 
{
	customers ONE 
	accounts MANY
}{};

association between 
{
	accounts ONE 
	contracts MANY
}{};

association between 
{
	contracts ONE 
	"life insurance" ONE
}{};

association between 
{
	contracts ONE 
	"real estate insurance" ONE
}{};