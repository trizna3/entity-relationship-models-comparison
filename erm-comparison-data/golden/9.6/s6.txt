account-accounts
contract-contracts
life insurance-life insurance
real estate insurance-real estate insurance
persons-customers
addresses-building address
real estate types-building type
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-building address-real estate-insurance
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-building type-real estate-insurance